swan-lake/learn-the-language/language-basics.md

```diff
@@ -227,7 +227,7 @@ int n = s.length();
 
 The ``substring()`` and ``length()`` functions are ``string`` lang library functions called using the convenient method call syntax. However, the functions are called on variables/values of the ``string`` type rather than objects.
 
-Ballerina imports the lang library for **``ballerina/lang.T``** where **``T``** represents a built-in type. Therefore, in the case of the above code example, you can also find the length of the string value by importing the **``ballerina/lang.string``** module and calling the **``length()``** function using the function call syntax.
+Ballerina imports the lang library for **``ballerina/lang.T``** where **``T``** represents a built-in type. Therefore, in the case of the above code example, you can also find the length of the string value by importing the **``ballerina/lang.'string``** module and calling the **``length()``** function using the function call syntax.
```

Comment:
I believe `ballerina/lang.string` is used here to refer to the `ballerina/lang.string` module itself rather than how it should be specified in the code when importing.

How about saying
```suggestion
Ballerina imports the lang library for **``ballerina/lang.T``** where **``T``** represents a built-in type. Therefore, in the case of the above code example, you can also find the length of the string value by importing the **``ballerina/lang.string``** module (**``import ballerina/lang.'string``**) and calling the **``length()``** function using the function call syntax.
```

Reply:
@MaryamZi Thank you. Yes while I was coding for a while I couldn't get the import right. This specification would be perfect.

Reply:
@season101, would you like to commit the suggestion so that we can incorporate the change in this PR itself?

Reply:
Also, please note that you don't have to explicitly import the `ballerina/lang.string` module to use it.

The following sample will also work.

```ballerina
public function main() {
    string str = "hello";
    int length = string:length(str);
}
```

This works because the `string` prefix is predeclared. See https://ballerina.io/spec/lang/master/#predeclared-prefix. 

We will update this section to include details about it too.

Reply:
@MaryamZi - Shall we create an issue to track updating this?

Reply:
The above doc update is done via https://github.com/ballerina-platform/ballerina-dev-website/pull/5422.